### PR TITLE
[F] Percy snapshots

### PR DESCRIPTION
**Pull request by viniciuskneves**
It adds Percy snapshots in the build process. It takes screenshots from every story so I added a param to skip the ones with playground and also some with dynamic data. My idea here is to not refactor all components to add screenshot capabilities.
I've also set the width of a specific component that has a different behavior for mobile only.

closes https://github.com/homeday-de/homeday-blocks/issues/226

Upgraded some dependencies to close dependabot PRs.

### CODE REVIEW

File: .travis.variables.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+if [ "$TRAVIS_BRANCH" = "master" ]
+  then
+    echo "Running PRODUCTION build"
+
+    export ENV=production
+    export PERCY_TARGET_BRANCH=master
+fi
+
+echo "ENV: $ENV"
+echo "PERCY_TARGET_BRANCH: $PERCY_TARGET_BRANCH"
```
Comment (viniciuskneves) [inline .travis.variables.sh]: This file is responsible for setting the `PERCY_TARGET_BRANCH` which is the branch that Percy will use as base for snapshots.

Comment (viniciuskneves) [reply .travis.variables.sh]: By default it is set do `develop` on Percy dashboard.

File: .travis.yml [hunk 2]
```diff
@@ -1,14 +1,26 @@
 language: node_js
 node_js:
-- 10.16.0
+- 10.17.0
```
Comment (viniciuskneves) [inline .travis.yml]: Latest v10 release (Shall we go to v12 by now?)

File: README.md [hunk 3]
```diff
@@ -1,4 +1,6 @@
-[![Build Status](https://travis-ci.com/homeday-de/homeday-blocks.svg?token=rxKqgsBkMR5FqNpvxBcx&branch=develop)](https://travis-ci.com/homeday-de/homeday-blocks) [![Coverage Status](https://coveralls.io/repos/github/homeday-de/homeday-blocks/badge.svg?t=Kbz7Vb)](https://coveralls.io/github/homeday-de/homeday-blocks)
+[![Build Status](https://travis-ci.com/homeday-de/homeday-blocks.svg?token=rxKqgsBkMR5FqNpvxBcx&branch=develop)](https://travis-ci.com/homeday-de/homeday-blocks)
+[![Coverage Status](https://coveralls.io/repos/github/homeday-de/homeday-blocks/badge.svg?t=Kbz7Vb)](https://coveralls.io/github/homeday-de/homeday-blocks)
+[![This project is using Percy.io for visual regression testing.](https://percy.io/static/images/percy-badge.svg)](https://percy.io/Homeday/homeday-blocks)
```
Comment (viniciuskneves) [inline README.md]: image

File: package.json [hunk 4]
```diff
@@ -21,13 +21,15 @@
...
     "lint": "vue-cli-service lint",
+    "lint:ci": "vue-cli-service lint --no-fix",
     "build:lib": "vue-cli-service build --target lib --name homeday-blocks ./src/components/index.js",
     "build:storybook": "vue-cli-service storybook:build -c config/storybook",
     "serve:storybook": "vue-cli-service storybook:serve -p 6006 -c config/storybook",
     "test:unit": "vue-cli-service test:unit",
     "test:unit:snapshot": "vue-cli-service test:unit -u",
     "test:unit:watch": "npm run test:unit -- --watch",
     "test:unit:ci": "npm run test:unit -- --ci --coverage",
+    "test:percy": "if [ $PERCY_TOKEN ]; then percy-storybook; else echo 'Skipping test: PERCY_TOKEN is not defined'; fi",
```
Comment (viniciuskneves) [inline package.json]: We need to check for `PERCY_TOKEN` otherwise it breaks as the variable can't be found.

**Comment by viniciuskneves** [comment]
> Looks good. 👍 I get the idea, is a bit hard to see what's skipped and what not from PR. Would it make sense to have a story and then one render(or how many are needed?) dedicated to Percy and rest we skip for screenshots? So we know to always add a case for Percy when writing stories. For that, I guess would be beneficial to reverse the condition and instead of `skip`, we would say `include`. :)

We can see how it grows for next stories. The rule of thumb is: have we rendered it before (previous story) and this new story is only changing its style --> We should skip, otherwise it should be rendered. I've skipped few of them so I think it is open for discussion. I will try to keep track of the next PR and see if it makes sense to reverse the logic and make it more strict (I do see the benefit of making it strict as it is the same of "writing tests" for that component, if you don't take screenshot it is "not adding tests").

Let's see if someone else adds a comment today otherwise I will move forward and see how it evolves.

Thanks for the feedback and thoughts.

**Comment by SinisaG** [comment]
> > Looks good. +1 I get the idea, is a bit hard to see what's skipped and what not from PR. Would it make sense to have a story and then one render(or how many are needed?) dedicated to Percy and rest we skip for screenshots? So we know to always add a case for Percy when writing stories. For that, I guess would be beneficial to reverse the condition and instead of `skip`, we would say `include`. :)
> 
> We can see how it grows for next stories. The rule of thumb is: have we rendered it before (previous story) and this new story is only changing its style --> We should skip, otherwise it should be rendered. I've skipped few of them so I think it is open for discussion. I will try to keep track of the next PR and see if it makes sense to reverse the logic and make it more strict (I do see the benefit of making it strict as it is the same of "writing tests" for that component, if you don't take screenshot it is "not adding tests").
> 
> Let's see if someone else adds a comment today otherwise I will move forward and see how it evolves.
> 
> Thanks for the feedback and thoughts.

Sounds good. :+1: Let's give it a try and see/learn.

**Pull request merged**